backend/api/whereaboutsApi.js

```diff
@@ -8,20 +8,20 @@ const whereaboutsApiFactory = client => {
   const put = (context, url, data) => client.put(context, url, data).then(processResponse())
 
   const getAttendance = (context, { agencyId, period, locationId, date }) =>
-    get(context, `/attendance/${agencyId}/${locationId}?date=${date}&period=${period}`)
+    get(context, `/attendances/${agencyId}/${locationId}?date=${date}&period=${period}`)
 
   const getAttendanceForBookings = (context, { agencyId, period, bookings, date }) =>
-    get(context, `/attendance/${agencyId}?${arrayToQueryString(bookings, 'bookings')}&date=${date}&period=${period}`)
+    get(context, `/attendances/${agencyId}?${arrayToQueryString(bookings, 'bookings')}&date=${date}&period=${period}`)
 
   const postAttendance = (context, body) => post(context, '/attendance', body)
 
   const putAttendance = (context, body, id) => put(context, `/attendance/${id}`, body)
 
-  const getAbsenceReasons = context => get(context, '/attendance/absence-reasons')
+  const getAbsenceReasons = context => get(context, '/absence-reasons')
 
   const getPrisonAttendance = (context, { agencyId, date, period }) =>
-    get(context, `/attendance/${agencyId}/attendance-for-scheduled-activities?date=${date}&period=${period}`)
-  const attendAll = (context, body) => post(context, '/attendance/attend-all', body)
+    get(context, `/attendances/${agencyId}/attendance-for-scheduled-activities?date=${date}&period=${period}`)
+  const attendAll = (context, body) => post(context, '/attendances', body)
```

Comment:
One for the next PR - It's not called attend-all now, create-attendances might be a better name.